screens/Authentication/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { View, StyleSheet, Text } from "react-native"
+import PerlinNoise from "@/components/PerlinNoise"
+import { CustomButton } from "@/components/ui/Button"
+import { theme } from "@/theme"
+import LogoIcon from "@/components/vectors/LogoIcon"
+import CustomText from "@/components/ui/CustomText"
+import { useSharedValue } from "react-native-reanimated"
+
+export default function EntryScreen() {
+  const handleCreateAccount = () => {
+    // this will navigate to create an account
+    // navigation.navigate("CreateAccount")
+  }
+
+  const handleSignIn = () => {
+    // to do this will navigate to the signinPage
+    // navigation.navigate("SignIn")
```

Comment:
can we have this navigate to the login and sign up screens in the meantime, so we can be able to navigate to other screens. While we are waiting for the actual login and sign up and sign up implementations?